scipy/optimize/_linprog.py

```diff
@@ -213,13 +217,17 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
         the corresponding element of ``b_eq``.
     bounds : sequence, optional
         A sequence of ``(min, max)`` pairs for each element in ``x``, defining
-        the minimum and maximum values of that decision variable. Use ``None`` to
-        indicate that there is no bound. By default, bounds are ``(0, None)``
-        (all decision variables are non-negative).
+        the minimum and maximum values of that decision variable. Use ``None``
+        to indicate that there is no bound. By default, bounds are
+        ``(0, None)`` (all decision variables are non-negative).
         If a single tuple ``(min, max)`` is provided, then ``min`` and
         ``max`` will serve as bounds for all decision variables.
-    method : {'interior-point', 'revised simplex', 'simplex'}, optional
+    method : {'highs-ds', 'highs-ipm', 'highs', 'interior-point',
+    'revised simplex', 'simplex'}, optional
```

Comment:
Indentation is wrong here. Anyway best to simply do `method : str, optional` here, because the options are already spelled out in the parameter description. An explicit listing is only better if there ~ <= 3 possible values.

Reply:
See https://github.com/mdhaber/scipy/pull/46; changed to simply `method : str, optional`